tests/golos.vesting_tests.cpp

```diff
@@ -584,7 +584,7 @@ BOOST_FIXTURE_TEST_CASE(delegate, golos_vesting_tester) try {
     const auto charge_prop = 0.7;
 
     BOOST_TEST_MESSAGE("--- fail when delegate to self");
-    BOOST_CHECK_EQUAL(err.unsuccessful_delegation, vest.msig_delegate(N(sania), N(sania), amount));
+    BOOST_REQUIRE_THROW(vest.msig_delegate(N(sania), N(sania), amount), fc::exception);
```

Comment:
actually using `msig_delegate` is wrong in this case. ordinary `vest.delegate` should be used: it adds 2 signatures and returns correct error message on fail